config/prod/ecmonsen-emorypipeline-lambda.yaml

```diff
@@ -11,10 +11,10 @@ parameters:
   # Duc Duong https://www.synapse.org/#!Profile:3320325
   # Eva Monsen https://www.synapse.org/#!Profile:3399376
   SynapseNotifyUsers: "3399376,3377637,3320325"
-  Department: SysBio
-  Project: amp-ad
-  OwnerEmail: kara.woo@sagebase.org
-  SynapseUserName: amp-ad-uploader
+  Department: "SysBio"
+  Project: "amp-ad"
+  OwnerEmail: "kara.woo@sagebase.org"
+  SynapseUserName: "amp-ad-uploader"
 hooks:
   before_launch:
-    - !cmd curl https://ecmonsen-emorypipeline-ci-lambdaartifactsbucket-jnlt9zs4haa7.s3.amazonaws.com/lambda-uploader/v1.0.3/ecmonsen-emorypipeline-lambda.yaml --create-dirs -o templates/remote/ecmonsen-emorypipeline-lambda.yaml
+    - !cmd "curl https://ecmonsen-emorypipeline-ci-lambdaartifactsbucket-jnlt9zs4haa7.s3.amazonaws.com/lambda-uploader/v1.0.3/ecmonsen-emorypipeline-lambda.yaml --create-dirs -o templates/remote/ecmonsen-emorypipeline-lambda.yaml"
```

Comment:
is the yamllint ok with this?  you may need to put quotes around the whole thing `- "!cmd curl.... -lamba.yaml"`